services/restund/README.md

```diff
@@ -127,3 +127,17 @@ Example rkt command:
     --user=restund \
     --group=restund
 ```
+
+In case You have set up restund without docker, you just need to make some of these changes:
+
+Put your private IP of the server in place of: `{{ ansible_default_ipv4.address }}`. And replace restund listen ports with `3478`, for both UDP and TCP.
```

Comment:
Note that private IP may be slightly misleading;

`turn_relay_addr` is the address that the TURN server will bind to
`turn_public_addr` may be the same or not; it is the address "over which other TURN servers should reach this TURN server over UDP". For a better description, check [this PR](https://github.com/wireapp/wire-server-deploy/pull/109)

Reply:
UDP for the server can be defined in the `udp_listen`. And `turn_public_addr` can be commented out if not needed.